app/javascript/article-form/components/Options.jsx

```diff
@@ -11,18 +11,44 @@ import CogIcon from '@images/cog.svg';
  * @param {Function} props.onSaveDraft Callback for when the post draft is saved
  * @param {Function} props.onConfigChange Callback for when the config options have changed
  */
+
+function toISOStringLocal(datetime) {
+  const month = (datetime.getMonth() + 1).toString().padStart(2, '0');
+  const day = datetime.getDate().toString().padStart(2, '0');
+  return [
+    datetime.getFullYear(),
+    '-',
+    month,
+    '-',
+    day,
+    'T',
+    datetime.toLocaleTimeString(),
+  ].join('');
+}
+
 export const Options = ({
   passedData: {
     published = false,
+    publishedAt = '',
+    timezone = Intl.DateTimeFormat().resolvedOptions().timeZone,
     allSeries = [],
     canonicalUrl = '',
     series = '',
   },
+  schedulingEnabled,
   onSaveDraft,
   onConfigChange,
 }) => {
   let publishedField = '';
   let existingSeries = '';
+  let publishedAtField = '';
+  const publishedDate = new Date(publishedAt);
+  const currentDate = new Date();
+
+  const localPublishedAt = toISOStringLocal(publishedDate);
+  const minPublishedAt = toISOStringLocal(currentDate);
+
+  const readonlyPublishedAt = published && publishedDate < currentDate;
```

Comment:
Maybe we just shouldn't show this option at all if the post is already published? Or if we do show it as readonly, add some description to explain it can't be edited as it's already published.